scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -101,6 +105,17 @@ internal fun generateSummary(
 ): ScanSummary {
     val header = result["headers"].single()
 
+    val issues = mutableListOf<Issue>()
+    val outputFormatVersion = header["output_format_version"]?.textValue()?.let { Semver(it) }
+    if (outputFormatVersion != null && outputFormatVersion > Semver(MAX_SUPPORTED_OUTPUT_FORMAT_VERSION)) {
+        issues += ScanCode.createAndLogIssue(
+            source = ScanCode.SCANNER_NAME,
+            message = "The output format version $outputFormatVersion exceeds the supported version " +
```

Comment:
Should we add ScanCode into the message, e.g. "The **ScanCode** output format version..."?

Reply:
I believe the `source` is sufficient here.

Reply:
The source cannot be matched  by resolutions...So, just from the message the scanner name cannot be matched. Anyhow, we also do not have this error for other scanners right now.

Reply:
BTW, this generally relates to https://github.com/oss-review-toolkit/ort/issues/4393.